### DISCUSSION TITLE
Hi-Res earing aids

### DISCUSSION THREAD

**Discussion by Manukatche** [m-r-s/hearingaid-prototype#38] [Q&A]
Hello people, 

I’m a french 41 yo man and I’m hard of hearing from my birth ( toxoplasmosis by congenital way ), see my audio profile: 
![Capture Audio PE 29092021](https://user-images.githubusercontent.com/114031902/191377391-45f63539-3707-4533-a706-ae4e31f2b75e.PNG)

 

Of course, I have to wear earing aids wich I show you the settings: 
![Capture Ampli PE 29092021](https://user-images.githubusercontent.com/114031902/191377433-7925a93b-98f5-4b49-8354-7a28168adb4a.PNG)

 

Since more than a year I listen to music with this hardware set : 
![Listen hardware](https://user-images.githubusercontent.com/114031902/191377466-d775f30d-ee78-4045-8736-52b66d7f8d2e.png)

 

I stream Hi-Res songs (24 bits, 192 kHz) from Qobuz trough a DAC wich drive 300 ohms headphones and  I use 21 ohms IEMs. I have to use 80% of DAC capacity amp in order to reach my earing cells. I guess you realize how much amp I need... 

I use parametric equalizer in order to match with my hearing loss but settings are not perfect cause I can’t use this EQ for each ear. Plus, I will soon make my audio profile with Earfull and Etymotic ER2SE custom for a better accuracy. See my EQ, trebbles are not set cause I crush the sound when I try to set. I dunno why... Seems no setting is the best trebble setting... 
![Parametric equalizer](https://user-images.githubusercontent.com/114031902/191377502-94c5bb89-9a37-45a5-bd56-bb0f9487418d.png)

 

 

However, this hardware set gives to me a heaven sound quality by powerfull amp with very low distorsions, large range frequencies ( 20 to 20 000 Hz ) and high resolution ( 24 bit, 192kHz and a lot of Kbps). 

Actually, I think I’m growing my earing neuronal web by assimilating frequencies I don’t ear with my conventional earings aids. I feel some connections are builded. Speeches become transcendants ^^ 

Why do I tell you that? You have probably already got me: I wanna earing aids with same quality sound such is provided by my hardware setup listening.  

I know I can’t really reach exactly the same quality sound cause microphone positioning but I wanna try to make earing aids whiches provide full frequencies range, high resolution sound and a parametric equalizer, I don’t like settings by “stick” equalizer. And I wanna be able to speak with someone who is at 5 meters to me ( optionnal but if it’s possible I take ).  

I'm lucky, I found your openMHA on Raspberry Pi project and I wanna build the same things but I have to consider my strong hearing loss which need a powerfull amp and there are somethings to custom in order to avoid feedbacks and to handle the huge data flow induced by Hi-Res. 

 

I have to complete this scheme : Capture, pre-amp, DCA, DSP, DAC, amp and reproduce. 

 

Capture and pre-amp: 

I would like to use Soundman OKM II microphones ( studio or classic version?) 

It's provided with a pre-amp wich have a jack output but I think it isn’tpossible to have true stereo sound coming from one jack. 
![Soundman OKM II classic with provided with pre amp](https://user-images.githubusercontent.com/114031902/191377546-1b01fbe9-ed7d-4764-adc0-95280bf1e477.png)

 

 

 https://www.thomann.de/gb/soundman_okmii_studio_solo.htm  (studio version) 

or 

https://www.thomann.de/intl/soundman_okmii_incl_adapter_a3.htm (classic version) 

 

If the soudman OKMII pre-amp won’t do the job, do I have to use this stereo pre-amp? 
![Stereo preamp](https://user-images.githubusercontent.com/114031902/191377573-d13e3c83-9abc-4318-88b8-36aedc9ce4d2.png)

 

https://www.kemo-electronic.de/en/Light-Sound/Amplifier-Splitter/Modules/M237-Stereo-Preamplifier.php  

 

DAC and amp: 

If I’m right, the Raspberry can’t provide enough electric power to supply my IEMs. 

This amp is also a DAC, I guess it’s a good choice. 
![FiiO Q3](https://user-images.githubusercontent.com/114031902/191377596-50b2e93f-28f7-4761-b270-f2bff100979f.png)

 

https://www.fiio.com/q3 

 

Reproduce: 

I will try my musical custom IEMs ( EA 502 ).  

But I think I will use the Etymotics ER2SE with custom tip cause it haves a flat response, wich is better for intelligibility. 
![EA 502](https://user-images.githubusercontent.com/114031902/191377616-377e9e69-c411-4002-9932-5ee48e4a40e1.png)

 

https://www.erdre-audio.com/in-ear-monitors/502/ (custom IEMs) 

Or 
![Etymotic ER2SE](https://user-images.githubusercontent.com/114031902/191377635-d0c860b0-6cbd-4aa9-a69f-29aee965c3eb.png)
![Custom tips ER2SE](https://user-images.githubusercontent.com/114031902/191377670-0c4a18cc-a9a3-439b-ab99-37a39d56d844.png)

                         

https://www.etymotic.com/product/er2se-earphones/ (custom tips: https://www.adv-sound.com/products/eartune-fidelity-custom-fit-eartips?variant=19293393158213 ) 

 

DCA and DSP: 

 

It’s here I need the Raspberry. Unfortunately, I’m a noob about programming and electronical. I wanna learn but I don’t know where I have to start, I have only some tiny knowledges by here, by low. 

I’m lost cause the Raspberry I need isn’t the same as the openMHA project describes.  

I guess I have to put a second USB-C, a lot of RAM and a crazy processor? 

But before to know how programming, I have to know whiches hardwares I need to build my Hi-Res earing aids. 

What I need: 

- Full frequencies range ( 20 to 20000 Hz ). I dunno what maximum frequency I’m able to hear ^^ I will know when I have my IEMs Etymotic ER2SE and do my audio profile on Earfull. 

- High resolution: 24 bit, 192 kHz and a lot of kbps. 

- Parametric equalizer for each ear. 

- A balance settings ( my left need more power than my right ). 

(optionnal) - Speak with someone who is at 5 meters ( By now, I can’t handle a conversation beyond 2 meters, my microphones are to small ). 

 

Of course, I’m curious about openMHA’s algorithms. I would like to try these. 

 

May you please help me to choose the hardwares I need? 

 

A big thx in advance

**Comment by m-r-s** [comment]
Dear Manukatche,

thank you for your contribution, report and questions.
I will first need to find time to answer in detail. But I will.

Best wishes,
Marc

**Reply by Manukatche** [reply]
Oooh thx for your involvement Marc ☺️

There was some progresses since I posted here.

I'm still waiting for my IEMs custom tips, I guess I will receive them 2 weeks from now. 

And, I will soon buy a SBD ADAU 1787, I think I can do something good with that 🧐
https://www.analog.com/en/products/adau1787.html#product-overview

So I'm still progressing I learn new things everyday.

I will be glad to read your advices Marc, thx again ☺️

**Reply by felipepimentab** [reply]
Hello Manukatche, thanks for sharing.
Have you made any progress with the ADAU 1787 since this post? Did you try porting open MHA to it?

**Comment by Zikovich** [comment]
Hi @Manukatche 

Thanks for this comprehensive sharing.

Can I ask if you can manage to purchase PHL "OpemMHA"?

Thanks in advance,
Islam,

**Comment by Methspirit** [comment]
Any idea for setting for a FiiO DAC Btr13?
